config/provider.go

```diff
@@ -21,7 +21,7 @@ type Kubernetes struct {
 	Delimiter string `mapstructure:"DELIMITER" yaml:"Delimiter" default:"_"`
 }
 
-var providers = []string{"docker", "docker_swarm", "swarm", "kubernetes"}
```

Comment:
Hi, fyi, I noticed docker_swarm was removed. Is this intentional?

Reply:
No, it should be added back